.gitignore

```diff
@@ -4,3 +4,5 @@ typings
 *.tgz
 .idea
 .tmp
+dist
+app/build
```

Comment:
it was supposed to work using http://rawgit.com, but not sure if it's still valid or not.